docs/resources/repository_access_rules.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
This file still needs some work. In particular, I need to create a template file with instructions on how the id is calculated, like we do for https://github.com/cyralinc/terraform-provider-cyral/blob/main/docs/resources/repository_identity_map.md. My understanding is that this file is somehow built from a template. I'll have to look into this.